packages/frontend/hooks/src/use-block-suite-page-preview.ts

```diff
@@ -25,14 +25,19 @@ export function useBlockSuitePagePreview(page: Page | null): Atom<string> {
   } else if (weakMap.has(page)) {
     return weakMap.get(page) as Atom<string>;
   } else {
-    const baseAtom = atom<string>(getPagePreviewText(page));
+    const baseAtom = atom<string>('');
```

Comment:
This is removed because `ready` slot will always be called?

Reply:
It will eventually set in `onMount`